src/main/python/pybuilder/plugins/python/sphinx_plugin.py

```diff
@@ -62,24 +99,41 @@ def assert_sphinx_is_available(logger):
 def sphinx_generate(project, logger):
     """Runs sphinx-build against rst sources for the given project.
     """
-    logger.info("Running sphinx-build")
-
-    log_file = project.expand_path("$dir_target/reports/{0}".format("sphinx-build"))
+    task_name = getattr(sphinx_generate, NAME_ATTRIBUTE)
     build_command = get_sphinx_build_command(project)
+    run_build(build_command, task_name, logger, project)
 
-    if project.get_property("verbose"):
-        logger.info(build_command)
 
-    exit_code = execute_command(build_command, log_file, shell=True)
-    if exit_code != 0:
-        raise BuildFailedException("Sphinx build command failed. See %s for details.", log_file)
+@task("sphinx_quickstart", "starts a new sphinx project")
+@depends("prepare")
+def sphinx_quickstart_generate(project, logger):
+    """Runs sphinx-build against rst sources for the given project.
+    """
+    task_name = getattr(sphinx_quickstart_generate, NAME_ATTRIBUTE)
+    build_command = get_sphinx_quickstart_command(project)
+    run_build(build_command, task_name, logger, project)
+
+
+def get_sphinx_quickstart_command(project):
+    """Builds the sphinx-quickstart command using project properties.
+        :param -q: Quiet mode that will skips interactive wizard to specify options.
```

Comment:
I think we need to make it clear that this documents the sphinx quickstart parameters.
Traditionally the docstring of a function would document the arguments taken by the function, so it should be made to stand out that this is not the case here.